app/build.gradle

```diff
@@ -277,6 +286,8 @@ dependencies {
     // Since java.time need API 26 we use that backport instead
     // See: https://github.com/JakeWharton/ThreeTenABP
     implementation 'com.jakewharton.threetenabp:threetenabp:1.3.0'
+
+    implementation 'com.google.re2j:re2j:1.6'
```

Comment:
Assuming that's the regular expression library used by Yuzu AdBlockPlus for some reason.

Reply:
Yes, from what I read re2j is faster than normal java regex parsing (at least for the expressions it understands) and used for some blocking filters.